package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-gettext-parser",
+  "name": "@elser/react-gettext-parser",
```

Comment:
Sorry, everything after commit e1af58f should not be excluded in the pull request.  should have made those changes on another branch. I'll resubmit the pull request on a different branch once you take a look at my replies to the other related comments.